src/Symfony/Component/Intl/Resources/data/currencies/en_150.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Version": "2.1.19.76",
+    "Names": {
+        "EUR": [
+            "€",
+            "Euro",
+            {}
```

Comment:
We don't seem to be using this value. There are occurrences of this third element in previous versions of icu data (see [here](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Intl/Resources/data/currencies/ca.json#L356-L360]) and [here](https://github.com/symfony/symfony/blob/65a8b9fc93cc05857f7865806110030980c1c8c2/src/Symfony/Component/Intl/Resources/data/currencies/ca.json#L356-L360)).

The original ICU [file stores](http://source.icu-project.org/repos/icu/icu/tags/release-57-1/source/data/curr/en_150.txt) some additional formatting definitions here:

```
en_150{
    %%Parent{"en_001"}
    Currencies{
        EUR{
            "€",
            "Euro",
            {
                "¤#,##0.00",
                ",",
                ".",
            }
        }
    }
    Version{"2.1.19.76"}
}
```

We seem to be removing it.